### PR TITLE
New versions for project templates

### PR DESCRIPTION
**Pull request by Ithildir**
Hi!
If CI completes correctly, can you please publish all `sdk/project-templates-*`, and also `sdk/project-templates` itself?

Thanks!!

cc @david-truong and @gamerson

### CODE REVIEW

File: modules/sdk/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java [hunk 1]
```diff
@@ -936,7 +936,7 @@ public void testBuildTemplateWorkspace() throws Exception {
 		_testNotExists(workspaceProjectDir, "wars/pom.xml");
 
 		_testContains(
-			workspaceProjectDir, "settings.gradle", "version: \"1.2.2\"");
+			workspaceProjectDir, "settings.gradle", "version: \"1.2.4\"");
```
Comment (gamerson) [inline modules/sdk/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]: Maybe we should remove this test so we aren't constantly haven't to bump it up.

Comment (gamerson) [inline modules/sdk/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]: Or just modify it so it isn't so fixed

Comment (Ithildir) [inline modules/sdk/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]: I agree! Once this pull is merged, I'll update the test 😊

**Comment by michaelhashimoto** [comment]
ci:stop

**Comment by michaelhashimoto** [comment]
ci:retest

**Comment by brianchandotcom** [comment]
ci:stop @Ithildir can you resend with the changes proposed by @gamerson ? This one took 5 hours for some reason.

**Comment by Ithildir** [comment]
ci:stop

**Pull request closed without merging**